packages/notebook/src/default-toolbar.ts

```diff
@@ -85,7 +85,8 @@ export namespace ToolbarItems {
           }
         });
       },
-      tooltip: 'Save the notebook contents and create checkpoint'
+      tooltip: 'Save the notebook contents and create checkpoint',
+      enabled: panel.context.contentsModel.writable
```

Comment:
Looking at [test failure](https://travis-ci.org/jupyterlab/jupyterlab/jobs/432703045#L1633) of `TypeError: Cannot read property 'writable' of null` maybe it is coming from here? Since there isn't all the `&&` checks?